etc/docker/tests/run_test.sh

```diff
@@ -7,5 +7,9 @@ cmake .. -G Ninja -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON \
 		-DVIAMCPPSDK_CLANG_TIDY=ON
 ninja all
 ninja install
-cd src/viam/sdk/tests
+pushd src/viam/sdk/tests
 UBSAN_OPTIONS="print_stacktrace=1" ctest --output-on-failure
+popd
+pushd src/viam/examples/modules/complex
+UBSAN_OPTIONS="print_stacktrace=1" ctest --output-on-failure
```

Comment:
I don't.. love this. I can try to make the `run_test.sh` a little smarter and/or copy the complex module test into the `src/viam/sdk/tests` directory. We have no precedence for tests being written outside of the `tests` directory, so wasn't quite sure what to do here.